bin/codecept.js

```diff
@@ -155,6 +156,20 @@ program.command('run-multiple [suites...]')
 
   .action(require('../lib/command/run-multiple'));
 
+program.command('info')
+  .description('Print debugging information concerning the local environment')
+  .action(() => {
+    console.log('\n Environment information:-\n');
+    envinfo
+      .run({
+        System: ['OS', 'CPU'],
+        Binaries: ['Node', 'Yarn', 'npm'],
+        Browsers: ['Chrome', 'Edge', 'Firefox', 'Safari'],
+        npmGlobalPackages: ['codeceptjs'],
```

Comment:
as @DavertMik aforementioned, may you add the `helpers` and `plugins` as well. Those info are also useful.

Reply:
Can you gimme a better picture? we have got the provision to use glob patterns to refer em.

Reply:
maybe showing smt like this:

```sh
   System:
    OS: Linux 5.0 Ubuntu 19.04 (Disco Dingo)
    CPU: (4) x64 Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz
  Binaries:
    Node: 8.11.4 - /usr/bin/node
    Yarn: 1.17.0-0 - /usr/local/bin/yarn
    npm: 6.10.3 - /usr/local/bin/npm
  Browsers:
    Chrome: 76.0.3809.87
    Firefox: 68.0.1
  npmGlobalPackages:
    codeceptjs: 2.2.1
 codeceptjsConfigs: {
  helpers: {
    Puppeteer: {
      url: 'https://www.pizza.de',
      windowSize: "1440x700",
      show: true
    },
    plugins: {
      allure: {
        outputDir: "report",
        enabled: true
     }
  }
```

@DavertMik may you expose more?